src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/VsLangProjectProperties.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using EnvDTE;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using VSLangProj;
+using VSLangProj110;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    internal class VsLangProjectProperties : VSProject, VSLangProj.ProjectProperties
```

Comment:
Partial class, by roslyn convention, would mean a nested type. So let me try to group the implementation separately without partial class. That way, it does not seem unorganized.